src/pnpq/devices/waveplate_thorlabs_k10cr1.py

```diff
@@ -331,16 +334,26 @@ def move_absolute(self, position: Quantity) -> None:
                 source=Address.HOST_CONTROLLER,
             ),
             lambda message: (
-                isinstance(message, AptMessage_MGMSG_MOT_MOVE_COMPLETED_20_BYTES)
-                and message.chan_ident == self._chan_ident
-                and message.position == absolute_distance
-                and message.destination == Address.HOST_CONTROLLER
-                and message.source == Address.GENERIC_USB
+                (
+                    isinstance(message, AptMessage_MGMSG_MOT_MOVE_COMPLETED_20_BYTES)
+                    and message.chan_ident == self._chan_ident
+                    and message.position == absolute_distance
+                    and message.destination == Address.HOST_CONTROLLER
+                    and message.source == Address.GENERIC_USB
+                )
+                or (
+                    isinstance(message, AptMessage_MGMSG_MOT_MOVE_COMPLETED_34_BYTES)
+                    and message.chan_ident_1 == self._chan_ident
+                    and message.chan_ident_2 == self._chan_ident
```

Comment:
I don't know if we can assume that `chan_ident_2` is also set to the real chan ident, and we really don't care -- we shouldn't try to match on it here.
```suggestion
```